helm/hpcc/templates/_helpers.tpl

```diff
@@ -928,7 +931,7 @@ Pass in dict with root and me
 {{- $serviceName := printf "sasha-%s" .me.name -}}
 {{- if .me.storage }}
 {{ include "hpcc.addVolume" (dict "root" .root "name" $serviceName "me" .me.storage) -}}
-{{ include "hpcc.addVolumes" (dict "root" .root "includeLabels" (list "sasha") ) }}
+{{ include "hpcc.addVolumes" (dict "root" .root "includeLabels" (list "sasha") "name" .me.storage.plane ) }}
```

Comment:
might this fail if no .me.storage dict ? i.e. does it need to have a $storage := .me.storage | default dict

Reply:
no, it works.  It sets it to a null value.

Reply:
(I tested with and without it set.  I need to add some test cases though...)